### PR TITLE
remove noalloc and fix fixed_seed setup

### PR DESCRIPTION
**Pull request by vanhauser-thc**
This removes the noalloc setup as it is not needed anymore.

It also fixed the fixed_seed setup to check how repeatable the tests are when all randomness is removed. After the next run the -s options can be removed from the aflplusplus setups.

Also the results of the aflplusplus_fortify are interesting and just need to run in the next run.

### CODE REVIEW

File: fuzzers/aflplusplus_mopt/fuzzer.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def fuzz(input_corpus, output_corpus, target_binary):
 
     flags = ['-L0']  # afl++ MOpt activation at once.
     flags += ['-prare']  # rare branch scheduling.
-    flags += ['-s666']  # fixed random seed.
+    flags += ['-s123']  # fixed random seed.
```
Comment (jonathanmetzman) [inline fuzzers/aflplusplus_mopt/fuzzer.py:43]: I'm not sure I understand why the seed is changing but I have no objections to doing so.

**Comment by jonathanmetzman** [comment]
I think you need to remove noalloc from CI.

File: fuzzers/aflplusplus_mopt/fuzzer.py (hunk 1 above)
Comment (vanhauser-thc) [reply fuzzers/aflplusplus_mopt/fuzzer.py:43]: just because then it is the same as in /aflplusplus/ - no other reason and not really helping or hurting :)

**Comment by vanhauser-thc** [comment]
> I think you need to remove noalloc from CI.

oh you are right! commited.

**Pull request merged**